core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -272,10 +272,11 @@ private void createSchemaTopic() throws StoreInitializationException {
   }
 
   static List<String> brokersToEndpoints(List<Broker> brokers) {
-    List<String> endpoints = new LinkedList<String>();
+    List<String> endpoints = new LinkedList<>();
     for (Broker broker : brokers) {
       for (EndPoint ep : JavaConversions.asJavaCollection(broker.endPoints())) {
-        endpoints.add(ep.connectionString());
+        String hostport = ep.host() == null ? ":" + ep.port() : Utils.formatAddress(ep.host(), ep.port());
+        endpoints.add(ep.securityProtocol() + "://" + hostport);
```

Comment:
Yeah, I forgot that on the client side it's a separate `security.protocol`. That's actually kind of annoying, not sure why we didn't use the same format as we have for `listeners` on the broker.

Which then just raises another question -- what should we do here if we encounter multiple security formats in ZK (which seems likely for anyone trying to enable security on their cluster during the transition period). I guess we could only use the first one and set `security.protocol` appropriately?